src/interfaces/api.ts

```diff
@@ -0,0 +1,10 @@
+export interface Products {
```

Comment:
This is a type declaration of a single Product, not multiple Products.
We usually declare interfaces for singular elements `Product`, and when used in plural form it will just be an Array of Product, so either `Product[]` or `Array<Product>`